### PR TITLE
bazel: update to 0.9.0

### PR DESCRIPTION
**Pull request by emcrisostomo**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix
- [x] update
###### Tested on

macOS 10.13.2
Xcode 9.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: devel/bazel/Portfile [hunk 1]
```diff
@@ -49,8 +49,10 @@ distname                  ${distname}-dist
 dist_subdir               ${name}
 
 # Checksum Phase
-checksums                 rmd160  5269d2fce917d3324dcb75ca06fe12089a4e24a1 \
-                          sha256  2418c619bdd44257a170b85b9d2ecb75def29e751b725e27186468ada2e009ea
+checksums                 rmd160  ed9c36b49d60aaead2d16bc28ece7c026745d632 \
+                          sha256  dfd0761e0b7e36c1d74c928ad986500c905be5ebcfbc29914d574af1db7218cf
+depends_build-append \
+    bin:bash:bash
```
Comment (l2dy) [inline devel/bazel/Portfile]: Why did you add this?

Comment (RootFunction) [reply devel/bazel/Portfile]: I didn't actually add this but I can see why @emcrisostomo did. The compile script needs to be run with the `bash` executable.

Comment (l2dy) [reply devel/bazel/Portfile]: macOS >= 10.5 should have bash with the system, so I think `bin:bash:bash` would be pointless.

Comment (emcrisostomo) [reply devel/bazel/Portfile]: @l2dy I'm afraid I lost a notification here. As @RootFunction pointed out, I added it because `bash` is required by the build infrastructure. I've been wondering whether or not to add that build dependency, and finally erred on the side of adding it. My first choice was actually adding a port dependency to `bash`, but I finally added `bin:bash:bash`. When writing that section I consulted the MacPorts documentation and found this:

> The most commonly-used type of dependencies in Portfiles are port dependencies, because dependencies should be provided by MacPorts ported software whenever possible, and usually only one port can provide the needed libraries and files.
>
> But when satisfying a dependency with vendor-supplied software is preferred for special reasons, or when it is possible for more than one port to satisfy a dependency, then file dependencies may be used. An example of the former is with ubiquitous utilities like awk, grep, make or sed, where OS X's versions are often sufficient; an example of the latter is with “-devel” ports—these ports provide a different version of the same files (though only one can be activated at a time).

So, if I understand that section correctly, even if the system provides a `bash` binary, it doesn't seem a sufficient reason _not_ to include a `bin:bash:bash` dependency. Maybe it's overkill, but this way the port is clear in stating its needs and is also more robust.

File: devel/bazel/Portfile [hunk 2]
```diff
@@ -49,8 +49,10 @@ distname                  ${distname}-dist
 dist_subdir               ${name}
 
 # Checksum Phase
-checksums                 rmd160  5269d2fce917d3324dcb75ca06fe12089a4e24a1 \
-                          sha256  2418c619bdd44257a170b85b9d2ecb75def29e751b725e27186468ada2e009ea
+checksums                 rmd160  ed9c36b49d60aaead2d16bc28ece7c026745d632 \
+                          sha256  dfd0761e0b7e36c1d74c928ad986500c905be5ebcfbc29914d574af1db7218cf
+depends_build-append \
```
Comment (l2dy) [inline devel/bazel/Portfile]: Minor nitpick, can you add an empty line before `depends_build-append`?

File: devel/bazel/Portfile (hunk 1 above)
Comment (l2dy) [reply devel/bazel/Portfile]: OK, I understand.

File: devel/bazel/Portfile (hunk 2 above)
Comment (emcrisostomo) [reply devel/bazel/Portfile]: Yes, done and squashed commits.

**Comment by kencu** [comment]
Please see this PR as well, which is also to update bazel to 0.8.1 https://github.com/macports/macports-ports/pull/1098

Current bazel version is up to 0.9.0, FYI, which I updated here based on #1098  <https://github.com/kencu/myports/blob/master/devel/bazel/Portfile>

**Comment by emcrisostomo** [comment]
Thanks @kencu, that's great. I actually wasn't bumping this PR again to 0.9.0 because it's open. I also see you've fixed the bash completion compilation, so we could probably close this and #1098 and use yours. If we do, are you filing a PR with your `Portfile`? Otherwise we can take yours and update either one of the existing PRs.

**Comment by kencu** [comment]
Oh, fine -- I should point out, though, that in #1098 there were some concerns about using ```exec``` and that will need to be changed to ```system -W``` (easy to do, just needs a test).

**Comment by kencu** [comment]
I should also say that perhaps we should close this PR out and let #1098 take it from here (perhaps with the update to 0.9.0) as that PR is from the maintainer of the port, I believe. What do you think?

**Comment by emcrisostomo** [comment]
@kencu, yes, I agree with you. Let's close this and let the maintainer take it from here. Thanks for the help.
